Replaced PNG logos with inline SVG logo.

webinista opened the pull request:
Closes department-of-veterans-affairs/vets.gov-team#615

- Used CSS to set logo color, hover states
- Fixed image rollover lag that was especially prominent on Playbook pages.

webinista committed:
Replaced PNG logos with inline SVG logo.

Closes department-of-veterans-affairs/vets.gov-team#615

- Used CSS to set logo color, hover states
- Fixed image rollover lag that was especially prominent on Playbook pages.

webinista committed:
Replaced PNG logos with inline SVG logo.

Closes department-of-veterans-affairs/vets.gov-team#615

- Used CSS to set logo color, hover states
- Fixed image rollover lag that was especially prominent on Playbook pages.

carlmjohnson commented on content/includes/header.html:
```diff
@@ -131,7 +131,11 @@
-      <h1 role="banner" class="va-header-logo"><a href="/">Vets.gov</a></h1>
+      <div role="banner" class="va-header-logo">
+        <a href="/">
+          <svg class="vets-logo" width="206" height="50" viewBox="0 0 206 50" xmlns="http://www.w3.org/2000/svg" pointer-events="none"><title>Vets dot gov from the U S Department of Veteran Affairs</title><g fill="none" fill-rule="evenodd"><g class="vets-logo-vetsdotgov"><path d="M194.04 37h3.84l6.88-19.44h-3.24l-3.68 11.04c-.56 1.92-1.2 3.88-1.76 5.72h-.16c-.6-1.84-1.24-3.8-1.8-5.72l-3.68-11.04h ...
```
Instead of having the SVG code mixed into HTML, can you use an include to pull it in?

webinista committed:
Moving SVG logo to separate file.

Including logo rather than using `object` or `img` because inline
SVG currently works better with assistive tech.

webinista committed:
Moving SVG logo to separate file.

Including logo rather than using `object` or `img` because inline
SVG currently works better with assistive tech.

jcmeloni-usds commented:
@webinista When you have a sec can you re-push to see if it kicks Jenkins into a green state (and then we can merge)? Thanks!

webinista committed:
Merge branch 'master' into issue-615-svglogo

webinista committed:
Resolving conflicts after merging master.

webinista committed:
Merge branch 'master' into issue-615-svglogo

webinista committed:
Resolving some regressions after merge.

webinista committed:
Merge branch 'master' into issue-615-svglogo

webinista committed:
Merge branch 'issue-615-svglogo' of github.com:department-of-veterans-affairs/vets-website into issue-615-svglogo

jcmeloni-usds commented:
@carlmjohnson You cool now with this? All checks passed and I'm apt to press the big green button.  Thanks!

carlmjohnson commented:
LGTM :shipit:
